bin/setup/README.md

```diff
@@ -0,0 +1,10 @@
+## How to Run?
```

Comment:
All this information should be moved to a CONTRIBUTING.md in the root of the project. `bin/setup` should just be a setup script with all the required steps to start developing to the project, and it can or not start the docker compose containers.